src/validate.js

```diff
@@ -1659,6 +1659,18 @@ function F_VALIDATE(disabledHL) {
 		var reverseSpeed = segment.$revMaxSpeed;
 		var forwardSpeedUnverified = segment.$fwdMaxSpeedUnverified;
 		var reverseSpeedUnverified = segment.$fwdMaxSpeedUnverified;
+		var speedConversionFactor = 1;  // default to metric
+		var speedMultipleFactor = 10;   // default to limits being set in multiples of 10
+		// check for MPH countries
+		if ((country == "United Kingdom") ||
+			(country == "Jersey") ||
+			(country == "Guernsey") ||
+			(country == "United States"))
+			speedConversionFactor = 1.609;
+		// countries with multiple of 5
+		if ((country == "United States") ||
+			(country == "Guernsey"))
+			speedMultipleFactor = 5;
```

Comment:
Validator should be configured per-country using `i18n`.

Reply:
Hmm, but the country is based on the segment being checked. And the translations are based on the user's localization. So when I edit in England for example, I can't rely on the translations. Those will be set to Dutch, while I am editing in England.
I agree with making this more dynamic, but we can't use the translation function for this I think.

Reply:
In Validator localization files work both for translation (based on language set in WME) and for localization (based on address of the segment).

So check params depend on country of the segment, see `validate.js`:

		var segment = new SimpleSEGMENT(segmentID);
		var address = segment.$address;
		var country = address.$country;

So `countryCode` depends on country of the segment if it's there. Otherwise we use a cached country code:

		var countryCode = country ? _I18n.getCountryCode(country.toUpperCase())
			: _RT.$cachedTopCCode;

And then the usage:

					options = getCheckOptions(52, countryCode);
					if (options[CO_NUMBER] < streetLen
						&& isLimitOk(52)
						&& address.isOkFor(52))
						segment.report(52);

Reply:
I'll make them configurable in the I18n for now, but it might be something to consider.